README.md

```diff
@@ -72,143 +75,29 @@ Directives from this repository are automatically tested with the following brow
 
-Each directive has its own AngularJS module without any dependencies on other modules or third-party JavaScript code. In practice it means that you can **just grab the code for the directives you need** and you are not obliged to drag the whole repository.
-
-### Quality and stability
-
-Directives should work. All the time and in all browsers. This is why all the directives have a comprehensive suite of unit tests. All the automated tests are executed on each checkin in several browsers: Chrome, ChromeCanary, Firefox, Opera, Safari, IE9.
-In fact we are fortunate enough to **benefit from the same testing infrastructure as AngularJS**!
-
-## Support
-
-If you are having problems making some directives work, there are several ways to get help:
+## Need help?
+Need help using UI Bootstrap?
 
 * Live help in the IRC (`#angularjs` channel at the `freenode` network). Use this [webchat](https://webchat.freenode.net/) or your own IRC client.
-* Ask a question in [stackoverflow](http://stackoverflow.com/) under the [angular-ui-bootstrap](http://stackoverflow.com/questions/tagged/angular-ui-bootstrap) tag.
-* Write your question in our [mailing list](https://groups.google.com/forum/#!categories/angular-ui/bootstrap).
-
-Project's issue on GitHub should be used discuss bugs and features.
-
-## Contributing to the project
-
-We are always looking for the quality contributions! Please check the [CONTRIBUTING.md](CONTRIBUTING.md) for the contribution guidelines.
-
-### Development
-#### Prepare your environment
-* Install [Node.js](http://nodejs.org/) and NPM (should come with)
-* Install global dev dependencies: `npm install -g grunt-cli karma`
-* Install local dev dependencies: `npm install` while current directory is bootstrap repo
-
-#### Build
-* Build the whole project: `grunt` - this will run `lint`, `test`, and `concat` targets
-* To build modules, first run `grunt html2js` then `grunt build:module1:module2...:moduleN`
-
-You can generate a custom build, containing only needed modules, from the project's homepage.
-Alternatively you can run local Grunt build from the command line and list needed modules as shown below:
+* Ask a question in [StackOverflow](http://stackoverflow.com/) under the [angular-ui-bootstrap](http://stackoverflow.com/questions/tagged/angular-ui-bootstrap) tag.
 
-```javascript
-grunt build:modal:tabs:alert:popover:dropdownToggle:buttons:progressbar
-```
-
-Check the Grunt build file for other tasks that are defined for this project.
-
-#### TDD
-* Run test: `grunt watch`
- 
-This will start Karma server and will continuously watch files in the project, executing tests upon every change.
+**Please do not create new issues in this repository to ask questions about using UI Bootstrap**
 
-#### Test coverage
-Add the `--coverage` option (e.g. `grunt test --coverage`, `grunt watch --coverage`) to see reports on the test coverage. These coverage reports are found in the coverage folder.
```

Comment:
@wesleycho I did not. I saved it in the Wiki and reference it from the readme:
https://github.com/angular-ui/bootstrap/wiki/Development